### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by joakim-hove**
Statoil/libres#392
Statoil/ert#153

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,14 +49,8 @@ will work:
 
     sudo pip install -r requirements.txt
 
-The Python + cmake interaction is handled in a separate project called
-[pycmake](https://github.com/Statoil/pycmake); you can either install
```
Comment (markusdregi) [inline README.md]: Should we not still mention `pycmake` in the README?

File: README.md [hunk 2]
```diff
@@ -49,14 +49,8 @@ will work:
...
-correct version of `pycmake` into your *libecl* code:
+The Python + cmake interaction is handled with two cmake modules in
+`python/cmake/Modules`. These modules will be installed under
+`$prefix/share/cmake/Modules` when you install `libecl`, and they will also be
+used by the downstream modules `libres` and `ert`.
 
-    git submodule update --init pycmake
-
-
-### Installing the Python code ###
```
Comment (markusdregi) [inline README.md]: How do you intend people to install the Python code? With `pip` or `setup.py` or the `CMAKE - enable python` flag?

Comment (joakim-hove) [reply README.md]: > How do you intend people to install the Python code? 

For now the `cmake --enable-python` is the by far best tested. At some point we should probably combine the `cmake + setup.py` based approaches. `pip install` will always only be a Python only solution - so if you are also installing `libres, ...` which utilizes the underlying C code directly, you can for the foreseeable future not use `pip`.

File: README.md (hunk 1 above)
Comment (joakim-hove) [reply README.md]: > Should we not still mention pycmake in the README? 

My suggestion is no; the problem is that `libres` and `ert` will use the `cmake` modules carried and installed by `libecl`; i.e. the dependency to these files is stronger than what is communicated through the use of a submodule.

The best thing would in my opinion be if we could deploy it to `pypi` - then we could add `pycmake` in the `requirements.txt` independently for `libecl, libres` and `ert` - but that requires some effort; so my suggestion is the current PR. No strong feelings though.

Comment (joakim-hove) [reply README.md]: There is no changes to the pycmake code, we could even duplicate it across the three repos. That way we simplify the build somewhat - I would actually vote for that.

Comment (andreabrambilla) [reply README.md]: minor comment: there is a quote charachter `"` after `build system`that is not matched anywhere

As was discussed earlier today, replicating the pycmake code can be a good option for the time being

File: README.md [hunk 3]
```diff
@@ -8,55 +8,53 @@ non-unified and formatted and unformatted files are supported.
...
-shared libraries built from the C code.
-
-You need to install some Python requirements before the Python code
-will work:
+### Alternative 2: Building with Python ###
+To build *libecl* with Python wrappers you just pass the option
+`-DENABLE_PYTHON=ON` when running `cmake`. In addition you need to install some
+dependencies first:
 
     sudo pip install -r requirements.txt
```
Comment (andreabrambilla) [inline README.md]: if we leave this line here, it's unclear when it has to be run (I know it's obvious if you know what you are doing, but let's make it as easy as possible).
I'd suggest to include it in the script below after the `cd libecl`

Comment (joakim-hove) [reply README.md]: OK - updated.

**Pull request merged**